components/navigable-container/index.js

```diff
@@ -124,18 +124,10 @@ export class NavigableMenu extends Component {
 		const eventToOffset = ( evt ) => {
 			const { keyCode } = evt;
 
-			const isVertical = orientation === 'vertical';
```

Comment:
I guess this means the "orientation" prop is useless now and we should remove all its references (docs, code...).
I'm not against this personally, we just have to make sure it's something we want for all `NavigableContainer` usage

Reply:
@jasmussen Are we ok with this change? It makes `left` and `up` / `down` and `right` work similarily in the inserter and the toolbar and the dropdowns?

Reply:
I removed it in d4df52a, but not sure if it is needed anywhere.

Reply:
Looking at the behavior in `master`, I do agree that it's a little weird that pressing _down_ goes _right_ (depending on where you are), whereas pressing _right_ does nothing. As such this _seems_ like a good change. But it also begs the question why down doesn't go down, and right doesn't go right. 

But it is the inserter, and there's a rich history in how we got here, so before I say anything definitive I'd love @afercia's thoughts.

Reply:
We've received some feedback during WCUS contributor day while testing the inserter, same confusion about _which_ arrows where expected to work. I think the arrow navigation should allow three "modes", depending on the nature of the UI component:
- horizontal: LEFT and RIGHT to go previous and next horizontally, e.g. toolbars
- vertical: UP and DOWN to go previous and next vertically, e.g. single column menus
- both: "spatial" navigation where UP and LEFT go previous, and DOWN and RIGHT go next

And yes, in the Inserter I'd use "both" 🙂

Reply:
@Soean Maybe instead of removing the "orientation" entirely, we keep it and add a third option "both" or "vertical-horizontal". That way we could enable this behavior in the inserter but still keep it as is for the BlockToolbar and the Dropdowns. The Dropdown should probably be only "vertical" while the block toolbar should be "horizontal". Thoughts?

Reply:
@youknowriad I had a similar thought. In WAI-ARIA 1.1 is a new default value `undefined: The element's orientation is unknown/ambiguous.` https://www.w3.org/TR/wai-aria-1.1/#aria-orientation